.travis.yml

```diff
@@ -9,13 +9,21 @@ addons:
       - ubuntu-toolchain-r-test
     packages:
       - g++-4.8
+      - python3-pip
 cache:
   directories:
     - node_modules
 before_script:
-  - pwd
-  - cp ~/build/HackIllinois/api/config/test.config.template ~/build/HackIllinois/api/config/test.config
   - npm install
+  - cp ./config/test.json.template ./config/test.json
+after_success:
+  - rm ./config/test.json
+before_deploy:
+  - ./scripts/install/flyway.sh
+  - ./scripts/install/deps.sh
+  - cp ./config/prod.json.template ./config/prod.json
+  - python3 ./scripts/deploy/deps.py
+  - npm run prod-migrations
 deploy:
   provider: elasticbeanstalk
   skip_cleanup: true
```

Comment:
We probably don't want this to be true, since there aren't any artifacts we want to keep from the build, and the prod config is copied over after cleanup.